BiweeklyMeetings/2022-07-19.md

```diff
@@ -0,0 +1,371 @@
+
+**Frederic** -: So one of the question I had regarding that entire process is, is Rob on board with it? So Rob is the original author from CIP 30 and the modification of Bayou.
+
+**Sebastien** -: Yeah.
+
+**Frederic** -: Okay.
+
+**Sebastien** -: And you can see in the discussion on the CIP, there's also Marcel from Farrell that's talked on this issue as well. The problem with the CIP is, as all other modifications to the CIP, there's no governance for CIP 30 that is explicit. So it's unclear when, if, how many changes to CIP 30 should be merged. So this PR, along with the next one we're talking after this, it's just stuck in this limbo. So to avoid getting things stuck in limbo, I just want to spend some time coming up with a way to change the CIP 30 structure to make it easier to have alternatives for various endpoints. And this is also something that the catalyst team was interested in, because they also want to use the same kind of flexibility to potentially introduce a governance API into CIP 30. So it may be that you want to merge this regardless, or we may want to keep this CIP on hold until we get this extended enabled functionality that, because the team also wants to be implemented, that will help their life and also help push this proposal through.
+
+**Frederic** -: I feel that merging will be the easiest path forward, but I agree that if this goes down that path and it can just become a bigger mess later, and it might warrant a separate conversation. By the way, for anybody interested in participating, there's a CIP chat room. I can see that Andrew's in the room right there. I don't know how to proceed regarding this one. Rob, how do you feel?
+
+**M. Àngels** -: Rob? I don't know if you are talking, but at least me, I am not able to listening to you.
+
+**Sebastien** -: He's just-
+
+**Frederic** -: I am hearing Rob.
+
+**M. Àngels** -: Okay.
+
+**Frederic** -: Maria's just restarting the stream. She'll be right back.
+
+**M. Àngels** -: You can refresh the page. Now I can hear you too, Rob.
+
+**Robert** -: Okay. Both of the options that Sebastian said sound good to me. I think it would be great to have some kind of governance process for this, because there are so many contributions. I think it would be great to include Rob on the governance question itself and then maybe go ahead with this or maybe wait, I don't have any particular opinion about it. Now I've lost everybody.
```

Comment:
```suggestion
**Robert** -: Okay. Both of the options that Sebastian said sound good to me. I think it would be great to have some kind of governance process for this, because there are so many contributions. I think it would be great to include Rob on the governance question itself and then maybe go ahead with this or maybe wait, I don't have any particular opinion about it. [Now I've lost everybody.]
```